.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+Before submitting a new issue, please check if a similar issue has already been filed.
+
+### Issue subject
+Briefly describe your issue here.
```

Comment:
Change `your` to `the` so that this won't make a bad or wrong impression that this is a 'personal' issue.